devnet/singularity/cluster_config.json

```diff
@@ -380,35 +380,35 @@
       "HOST": "127.0.0.1",
       "PORT": 38000,
       "ID": "S0",
-      "CHAIN_MASK_LIST": [
-        4
+      "FULL_SHARD_ID_LIST": [
+        1, 262145
```

Comment:
I would suggest using hex string like `"0x1"`, `"0x00010001"` then convert to integers when parsing the config

Reply:
Done.